Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+WORKDIR /usr/src/databricks-cli
+
+COPY . .
```

Comment:
nit: would it be more conventional to specify the target path absolutely?

Reply:
Considering WORKDIR instruction above, I don't think so.